libs/game/game.ts

```diff
@@ -158,7 +158,7 @@ namespace game {
      * Finish the game and display the score
      */
     //% group="Gameplay"
-    //% blockId=gameOver block="game over || %win=toggleWinLose with %effect effect"
+    //% blockId=gameOver block="game over %gameWon=toggleWinLose || with %effect effect"
```

Comment:
can we keep %win ?

Reply:
Yup, undid that already, just need to fix upgrade rules to handle it appropriately